packages/sdks/javascript/package.json

```
@@ -13,7 +13,10 @@
   "scripts": {
     "build": "rm -Rf ./dist && npm run lint && tsc",
     "build-api-models": "rm -Rf ./src/api/generated && JAVA_OPTS=\"${JAVA_OPTS} -Dlog.level=warn\" openapi-generator-cli generate && npm run build",
-    "docs": "typedoc",
+    "docs-remove-service-index": "rm -f src/services/index.ts",
+    "docs-export-service-index": "find src/services/ ! -name 'index.ts' -name '*.ts' -type f -exec bash -c 'echo \"export * from $(echo -e \"\\x27\")./$(basename $0 .ts)$(echo -e \"\\x27\")\" >> src/services/index.ts' {} \\;",
```

If you think it's a good idea to sort the contents in the `index.ts` file, you can change the command like this:

```suggestion
    "docs-export-service-index": "find src/services/ ! -name 'index.ts' -name '*.ts' -type f | sort -k3 -t'/' | xargs -I'{}' bash -c 'echo \"export * from $(echo -e \"\\x27\")./$(basename $0 .ts)$(echo -e \"\\x27\")\" >> src/services/index.ts' {} \\;",
```